DesignSafe-CI/portal
# Task/DES-1708(1714, 1581) - Project Overview Updates and updated PII Policy Guidelines

The **Project Overview** appears after creating a new project, editing a project's type, or visiting the project overview.

### Changes:
1. Updated the User Guidelines for storing and publishing Protected Data (PII)
2. Added additional overview slides for Field Research
3. Added task to generate notification email for project admins when PII is going to be published (part of overview)
4. Fix various bugs in the UI which prevented the Project Overview from appearing in the Working Directory and Curation Directory

### Testing:
1. Open an empty project (or one you do not care about completely erasing).
2. Click **Edit Project** at the top of the page.
3. Click **Edit Project Type** near the top of the modal. (this will load the Project Overview)
4. Select a radio button for a project type (note: Hybrid Simulation and Other do not have Overviews currently and should skip directly to the Best Practices page)
5. Check transition of slides...
6. Go back to the start and select the radio button for **Field Research**
7. There should be updated slides for the project type. Near the end you will reach another slide with radio buttons. This page will save the user's selection, and generate an email for the project admins when a user finishes and updates their project type to a **Field Research** model

This email be triggered when a user is selecting a project type for the first time or changing their project to a Field Research project that will have personally indefinable information (ex: clicking any of the options that say **yes**.)

To test, you can change the emails in **common_settings** to your own. Go into a new or empty project (or one you don't mind erasing) and open the "Edit Project" modal. From there click "Change Project Type" select "Field Research" and continue through the slides. You'll reach a slide with radio buttons and 3 choices any choices that say "YES" should send the email. Selecting "No" should allow the user to continue without an email being sent.

![Screen Shot 2020-07-27 at 5 31 56 PM](https://user-images.githubusercontent.com/29575979/88598538-1ace7600-d02f-11ea-8414-8921a340f17b.png)

8. Go to the Working Directory and Curation Directories and check that the **View Overview** hyperlink works (next to project type at the top of the projects area)
9. Loading the Project Overview like this should allow you to "preview" the project type's overview slides without being able to go back to the choices of selecting a project type or skip to the end.
